packages/web/src/ui/views/shortcuts.js

```diff
@@ -7,7 +7,7 @@ module.exports = function shortcuts (state, i18n) {
 
     // if we are in the midst of assigning a keypress (inProgress) display 'type & hit enter'
     // if we already have a temporary key use that one instead
-    const placeholder = (tmpKey && tmpKey.length > 0) ? tmpKey : i18n.translate('type & hit enter')
+    const placeholder = (tmpKey && tmpKey.length > 0) ? tmpKey : i18n.translate('typeandenter')
```

Comment:
same question as previously, to me default translation 'keys' where mean to be human centric , and not 'variables' but I might have been wrong, got very little experience

Reply:
I’ve seen both. But as soon as you add three options (previously two) then the default ‘key’ becomes meaningless. And, you have to change both code and locale files. This is a HUGE problem for maintenance, and always causes mistakes.

CONTEXT is more important for the ‘keys’

Reply:
ok. let's compromise a little.

the keys (strings) inside the code should be very simple:
- short as possible english phrases
- no special characters (parens, exclamations, etc)

the locales (key value pairs) can contain anything
- the best phrase for translation
- can include parens, exclamations, etc